pandas/core/groupby/ops.py

```diff
@@ -425,7 +429,13 @@ def _get_cython_func_and_vals(
         return func, values
 
     def _cython_operation(
-        self, kind: str, values, how: str, axis, min_count: int = -1, **kwargs
+        self,
+        kind: str,
+        values: np.ndarray,
```

Comment:
below we have checks for categorical_dtype, so either this annotation is wrong or we can remove some checks

Reply:
Yes it's wrong, probably `ArrayLike`?

Reply:
Nevermind mypy doesn't like that, will remove entirely for now